filesystem/btrfs_stream.ksy

```diff
@@ -0,0 +1,151 @@
+meta:
+  id: btrfs_stream
+  application: Btrfs
+  file-extension: btrfs-stream
+  endian: le
+  license: CC0-1.0
+doc: |
+  This spec can be used to disassemble raw Btrfs Send stream format
+  version 1 from the output of `btrfs send` command.
+  See also the `btrfs receive --dump` command.
+doc-ref: https://btrfs.wiki.kernel.org/index.php/Design_notes_on_Send/Receive
+
+seq:
+  - id: header
+    type: send_stream_header
+  - id: commands
+    type: send_command
+    repeat: eos
+types:
+  send_stream_header:
+    seq:
+      - id: magic
+        contents: ['btrfs-stream', 0x00]
+      - id: version
+        type: u4le
+  send_command:
+    seq:
+      - id: size
```

Comment:
Given that this attribute is solely used to define size of other field in bytes, the [recommended naming per style guide](https://doc.kaitai.io/ksy_style_guide.html#attr-id) is `len_data`:

```suggestion
      - id: len_data
```

Reply:
Replaced size by len_data